app/src/main/java/com/example/dynamicdiet/dao/IDietDao.kt

```diff
@@ -1,5 +1,5 @@
-interface ICountryDao {
-    fun getDietInfo(): Diet {
+interface IDietDao { // Changed it from ICountryDao to IDietDao.
```

Comment:
Changed the interface, and removed : Diet. This removes the error that it throws